packages/protocol/test-sol/common/Proxy.t.sol

```diff
@@ -0,0 +1,204 @@
+pragma solidity ^0.5.13;
+
+import "@celo-contracts/common/Proxy.sol";
+
+import "celo-foundry/Test.sol";
+
+import { Constants } from "@test-sol/constants.sol";
+import { Utils } from "@test-sol/utils.sol";
+
+import "@celo-contracts/common/test/GetSetV0.sol";
+import "@celo-contracts/common/test/GetSetV1.sol";
+import "@celo-contracts/common/test/HasInitializer.sol";
+import "@celo-contracts/common/test/MsgSenderCheck.sol";
+
+contract ProxyTest is Test, Constants, Utils {
+  Proxy proxy;
+  GetSetV0 getSet;
+  GetSetV0 proxiedGetSet;
+
+  event ImplementationSet(address indexed implementation);
+  event OwnerSet(address indexed owner);
+
+  function setUp() public {
+    proxy = new Proxy();
+    getSet = new GetSetV0();
+    proxiedGetSet = new GetSetV0();
+  }
+}
+
+contract ProxyTest_getOwner is ProxyTest {
```

Comment:
Follow [best practice/convention](https://book.getfoundry.sh/tutorials/best-practices#general-contract-guidance) when naming test contracts here and throughout